packages/plugins/src/hardhat/tasks.ts

```diff
@@ -30,8 +32,16 @@ const TASK_CHUGSPLASH_BUNDLE_LOCAL = 'chugsplash-bundle-local'
 const TASK_CHUGSPLASH_BUNDLE_REMOTE = 'chugsplash-bundle-remote'
 
 // public tasks
+const TASK_CHUGSPLASH_INIT = 'chugsplash-init'
+const TASK_CHUGSPLASH_LIST_ALL_PROJECTS = 'chugsplash-list-all-projects'
 const TASK_CHUGSPLASH_VERIFY = 'chugsplash-verify'
 const TASK_CHUGSPLASH_COMMIT = 'chugsplash-commit'
+const TASK_CHUGSPLASH_PROPOSE = 'chugsplash-propose'
+const TASK_CHUGSPLASH_APPROVE = 'chugsplash-approve'
+const TASK_CHUGSPLASH_LIST_PROPOSALS = 'chugsplash-list-proposals'
+
+// This address was generated using Create2 so it shouldn't need to be changed for now
+const CHUGSPLASH_REGISTRY_ADDRESS = '0x236151FE7938Fc16600a894E8c40C2E862503D3b'
```

Comment:
We can probably consider exporting this in `@chugsplash/contracts` so the address automatically updates if we when we change the source during development

Reply:
Yeah, I'll leave it as-is in this PR and one of us (or a new contributor!) can tackle the separate issue that you created